#VRML_SIM R2019a utf8
WorldInfo {
}
Viewpoint {
  orientation -0.0655303955212041 -0.9960549813566711 -0.05983511826173452 1.4839372338301184
  position -0.9102210930693081 0.40010330695978846 -0.09970734153600058
  follow "robot"
  followOrientation TRUE
}
TexturedBackground {
}
TexturedBackgroundLight {
}
Floor {
  size 100 100
}
Robot {
  translation 0 0.0799999921645342 0
  children [
    DEF bot Group {
      children [
        Transform {
          translation 0.1075 -0.0575 0
          rotation 1 0 0 1.5707963220000003
          children [
            Shape {
              appearance Appearance {
                material Material {
                  diffuseColor 0 1 0.5
                }
              }
              geometry Sphere {
                radius 0.0175
              }
            }
          ]
          rotationStep 0
        }
        DEF Caja Shape {
          appearance DEF Color_1 Appearance {
            material Material {
              diffuseColor 0 0.5 1
            }
          }
          geometry Box {
            size 0.25 0.08 0.1
          }
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 0.065
      }
      device [
        PositionSensor {
          name "position_sensor_right"
          resolution 0.0052
        }
        RotationalMotor {
          name "motor_right"
          consumptionFactor 300
          maxVelocity 52.3598775
          maxTorque 0.593170353999997
        }
      ]
      endPoint Solid {
        translation -0.05 0 0.065
        rotation 1 0 0 1.5707963220000003
        children [
          DEF Wheel Shape {
            appearance Appearance {
              material Material {
                diffuseColor 1 0 0
              }
            }
            geometry Cylinder {
              height 0.03
              radius 0.075
            }
          }
        ]
        boundingObject USE Wheel
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 -0.065
      }
      device [
        PositionSensor {
          name "position_sensor_left"
          resolution 0.0052
        }
        RotationalMotor {
          name "motor_left"
          consumptionFactor 300
          maxVelocity 52.3598775
          maxTorque 0.593170353999997
        }
      ]
      endPoint Solid {
        translation -0.05 0 -0.065
        rotation 1 0 0 1.5707963220000003
        children [
          USE Wheel
        ]
        name "solid(1)"
        boundingObject USE Wheel
        physics Physics {
        }
      }
    }
    DistanceSensor {
      translation 0.125 0 0
      children [
        Shape {
          geometry Box {
            size 0.01 0.01 0.01
          }
        }
      ]
      name "ds"
      lookupTable [
        0 0 0
        0.02 0 0
        5 1660 0
      ]
      type "sonar"
      resolution 1
    }
  ]
  boundingObject USE bot
  physics Physics {
  }
  controller "Crazybot"
}
